ofl/redditsans/METADATA.pb

```diff
@@ -0,0 +1,104 @@
+name: "Reddit Sans"
+designer: "Stephen Hutchings"
```

Comment:
@eliheuer following their copyright string, Reddit Inc probably needs to be credited too